README.md

```diff
@@ -8,35 +8,31 @@ You can create a YAML specification of your entities that you can pass directly
 rails g frontier_scaffold /path/to/yml
 ```
 
-## Supported Data-Types
+Frontier Generators also provides unit and features specs where applicable.
```

Comment:
I feel like it should read
`Frontier Generators also provides units and features specs where applicable.`
or
`Frontier Generators also provides unit and feature specs where applicable.`